src/review/index.ts

```diff
@@ -3,27 +3,33 @@ import { constructPromptsArray } from "./prompt/constructPrompt";
 import { getFileNames } from "./prompt/getFileNames";
 import { getMaxPromptLength } from "../common/model/getMaxPromptLength";
 import { commentOnPR } from "../common/ci/commentOnPR";
+import { commentPerFile } from "../common/ci/commentPerFile";
 import { signOff } from "./constants";
 
 interface ReviewArgs {
   [x: string]: unknown;
   ci: boolean;
+  commentPerFile: boolean;
   _: (string | number)[];
   $0: string;
 }
 
 export const review = async (yargs: ReviewArgs) => {
   const isCi = yargs.ci;
+  const isLineByLine = yargs.commentPerFile;
```

Comment:
Maybe we can rename `isLineByLine` into something referring to commenting per file since we're doing it by file now

```suggestion
  const shouldCommentByFile = yargs.commentPerFile;
```